events/class_tests/meeting_tests.py

```diff
@@ -59,3 +67,15 @@ def test_persist_possible_meeting_in_db(self):
     def test_duplicate_possible_meeting(self, persist_possible_meeting):
         with pytest.raises(ValidationError, match='meeting hours already exists'):
             persist_possible_meeting.save()
+
+    def test_get_all_event_dates(self, expected_meeting_results, event_title="event3"):
+        event = Event.objects.get(title=event_title)
+
+        assert expected_meeting_results == list(PossibleMeeting.objects.get_all_event_dates(event))
+
+    def test_remove_all_possible_dates(self, expected_meeting_results, event_title="event3"):
+        event = Event.objects.get(title=event_title)
+
+        PossibleMeeting.objects.remove_all_possible_dates(event)
+
+        assert expected_meeting_results not in list(PossibleMeeting.objects.all())
```

Comment:
please remove blank lines
here we have too many
```suggestion
        event = Event.objects.get(title=event_title)
        PossibleMeeting.objects.remove_all_possible_dates(event)
        assert expected_meeting_results not in list(PossibleMeeting.objects.all())
```

Reply:
done. thanks 😄